app/src/backlight.c

```diff
@@ -42,56 +39,41 @@ static struct backlight_state state = {.brightness = CONFIG_ZMK_BACKLIGHT_BRT_ST
 
 static int zmk_backlight_update() {
     uint8_t brt = state.on ? state.brightness : 0;
-    for (int i = 0; i < BACKLIGHT_NUM_LEDS; i++) {
-        int rc = led_set_brightness(backlight_dev, i, brt);
```

Comment:
This section is required if you have multiple LEDs, see [https://deploy-preview-904--zmk.netlify.app/docs/features/backlight#multiple-backlight-leds](https://deploy-preview-904--zmk.netlify.app/docs/features/backlight#multiple-backlight-leds).
What is the reason for removing it?

Reply:
Ah I see i didn't understand the need for multiple LEDS when it's all driven through one channel, will add back